java/java10.yaml

```diff
@@ -36,3 +36,6 @@ commandTests:
 - name: "xmllint tool"
   command: ["xmllint", "--version"]
   expectedError: ["using libxml version"]
+- name: "docker"
+  command: ["docker", "--version"]
+  expectedOutput: ["Docker version * "]
```

Comment:
nit: this may not like the trailing space in the quotes

Also, please restore the trailing newline.

Reply:
done